.github/resources/manifests/argo/overlays/tls-enabled/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../../../../manifests/kustomize/env/cert-manager/platform-agnostic-multi-user-tls
```

Comment:
We don't have any existing GitHub workflows deployed in multi-user mode (some are being added) so if you encounter trouble, you can also create an additional kustomize directory for standalone mode and use that for testing.